app/views/admin/resources/index.html.haml

```diff
@@ -20,17 +22,16 @@
                 = link_to(admin_conference_resource_path(@conference.short_title, resource.id)) do
                   = resource.name
                 %td
-                  = quantity_left_of(resource)
+                  %span{ id: "update_actions_#{resource.id}" }
+                    = render partial: 'update_actions', locals: { conference: @conference, resource: resource }
                 %td
-                  = resource.used
+                  %span{ id: "resource_used_#{resource.id}" }
+                    = resource.used
                 %td
                   .btn-group
                     = link_to 'Edit', edit_admin_conference_resource_path(@conference.short_title, resource.id),
                       method: :get, class: 'btn btn-primary', disabled: !(can? :update, resource)
                     = link_to 'Delete', admin_conference_resource_path(@conference.short_title, resource.id),
                       method: :delete, class: 'btn btn-danger', disabled: !(can? :destroy, resource),
                       data: { confirm: "Do you really want to delete #{resource.name}?" }
-.row
```

Comment:
No, it is **not** appropriate if the 'New' button is sometimes on top, and sometimes on the bottom. This is not good for our design, not helpful to our users. You need to put where users can find it, based on their experience from other parts of the application. 
If you disagree with its position, that's a different issue you can bring up, and if decided so, it would change for the whole application, not just 1 page. Makes sense?